packages/tests/helpers/index.ts

```diff
@@ -63,9 +64,9 @@ export const getDatabaseCredentials = () => {
         config.user = process.env.DATABASE_USER || 'postgres'
         config.password = process.env.DATABASE_PASSWORD || 'postgres'
     } else {
-        config.dbName = process.env.DATABASE_NAME || 'tensei'
+        config.dbName = process.env.DATABASE_NAME || 'mikrotensei'
```

Comment:
Please revert. This is used on GITHUB ACTIONS. Use your own environment variables locally by running `export DATABASE_NAME=mikrotensei` in the terminal before running your tests.